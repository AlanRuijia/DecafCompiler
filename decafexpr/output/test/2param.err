output prefix: /Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param
llvmas: /usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llvm-as
llc: /usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llc
cc: gcc
codegen: /Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decafexpr
stdlib: /Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decaf-stdlib.c
generating llvm code:/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decafexpr ... ok
; ModuleID = 'C'

define i32 @main() {
entry:
  ret i32 0
}
assembling to bitcode:/usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llvm-as "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param.llvm" -o "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param.llvm.bc" ... ok
converting to native code:/usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llc "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param.llvm.bc" -o "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param.llvm.s" ... ok
linking:gcc -o "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param.llvm.exec" "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param.llvm.s" "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decaf-stdlib.c" ... ok
running:/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2param.llvm.exec ... ok
