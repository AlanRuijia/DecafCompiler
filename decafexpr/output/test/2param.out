Debug message: Generating extern function...
Arg Type: 17 
Inserting method decl into symbol table...
Inserting block into symbol table...
Inserting method decl into symbol table...
Inserting block into symbol table...
Debug message: Generating program...
Debug message: Generating package...
Debug message: Generating FieldDecls...
Debug message: Generating MethodDecl...
Defind variable in line 3 : x
Defind variable in line 3 : y
Debug message: Generating method call... Calling print_int
ArgumentListSize = 1
Argument Type: Codegen for RValue x
Success in RValue
